pkgs/cloudflare/cfssl/cfssl/registry.yaml

```diff
@@ -1,5 +1,6 @@
 packages:
   - type: github_release
+    name: cloudflare/cfssl/cfssl
```

Comment:
```suggestion
    name: cloudflare/cfssl/cfssl
    aliases:
      - name: cloudflare/cfssl
```

Please set `aliases` to keep the compatibility.

Reply:
resolved https://github.com/aquaproj/aqua-registry/pull/15605/commits/f695648d54a4dedd3548873e28d72f80bfe5a456